lib/dice_bag/private_key.rb

```diff
@@ -25,6 +25,14 @@ def to_rsa_format!
       @private_key = [HEADER, body, FOOTER].flatten.join("\n")
     end
 
+    def public_key
+      rsa_object.public_key
+    end
+
+    def rsa_object
+      @rsa_object ||= OpenSSL::PKey::RSA.new @private_key
```

Comment:
IMO `new(@private_key)` would read better here now you're assiging it (before I'd have use `()` too but it wasn't so bad. Now with assignment it reads a tad weird)

Same goes for the other occurrences in the other files below.

Reply:
sounds like this project needs rubocop on CI